course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
Don't we want to wait to set this to false until the http calls we triggered are finished?